suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -658,6 +658,20 @@ export const getTestnetSymbols = () => getTestnets().map(n => n.symbol);
 export const isBlockbookBasedNetwork = (symbol: NetworkSymbol) =>
     networks[symbol]?.customBackends.some(backend => backend === 'blockbook');
 
+export const isDebugOnlyAccountType = (
+    symbol?: NetworkSymbol,
+    accountType?: NonNullable<AccountType | undefined>,
+): boolean => {
+    if (!symbol) return false;
```

Comment:
use rules from code style guide https://www.notion.so/satoshilabs/Basic-syntax-49331c82e0d14fb7a1229b6460009e64 (spacings and also you can add bracets